kubernetes/overlays/notifier/configmap.yaml

```diff
@@ -21,3 +21,9 @@ data:
   INTERNAL_JWT_SUBJECT: "{{INTERNAL_JWT_SUBJECT}}"
   INTERNAL_JWT_AUDIENCE: "{{INTERNAL_JWT_AUDIENCE}}"
   INTERNAL_JWT_DURATION_SECONDS: "{{INTERNAL_JWT_DURATION_SECONDS}}"
+
+  NOTIFICATION_POSTGRES_TABLE_NAME: "{{NOTIFICATION_POSTGRES_TABLE_NAME}}"
+  NOTIFICATION_POSTGRES_SCHEMA_NAME: "{{NAMESPACE}}_keys_notifications"
```

Comment:
The schema name could be generic and maybe in the future include more notifications?